src/channel.ts

```diff
@@ -1084,7 +1084,7 @@ export class Channel<StreamChatGenerics extends ExtendableGenerics = DefaultGene
 
   /**
    * removes the hidden status for a channel
-   *
+   * Has to be called before Channel.sendMessage() in order WS event `channel.visible` is emitted.
```

Comment:
I don't understand this comment, backend will automatically mark a channel as visible when we send a new message. This might be required some SDK workaround, but in that case that should live in an SDK specific documentation.

Reply:
I am also in favour of it and talked about it with @MartinCupela. The BE team cannot pick this up right now because of the lack of resources I think.

Reply:
This whole thing should ideally happen on the backend.

Reply:
@szuperaz Backend marks the channel as visible with a new message, but does not send the channel.visible event unless you explicitly call `channel.show()` before `channel.sendMessage()`. If we want to keep it in the SDK docs, then all the JS SDKs should add this not into the documentation. I can remove the comment. Please confirm, if you insist it should not be mentioned here.

Reply:
It doesn't relevant to all JS SDKs, for example Angular will show the channel even without calling `channel.show()`, so I think it's misleading to have it in the client. Also,the comment doesn't mention what is the problem we solve with calling `channel.show()` so I don't really think an integrator would understand the comment on their own anyway, I'm also not sure how would they find that comment in the code

Reply:
@szuperaz I have removed the doc string

Reply:
Thank you